scapy/route.py

```diff
@@ -8,7 +8,7 @@
 """
 
 import socket
-from scapy.arch.consts import LOOPBACK_NAME
+from scapy.arch.consts import LOOPBACK_NAME, WINDOWS
```

Comment:
Oh i remember: it was in [this](https://github.com/secdev/scapy/pull/378/commits/2f04cbf737f2c86a51a50fb2141882a2f606ced7) (in another fix) and for some reason the WINDOWS was never imported... It's a tiny thing... should i remove it ?